RDKShell/RDKShell.cpp

```diff
@@ -4951,7 +4965,21 @@ namespace WPEFramework {
         {
             bool ret = false;
             lockRdkShellMutex();
+             std::vector<std::string> clientList;
+            CompositorController::getClients(clientList);
+            bool targetFound = false;
+            for (size_t i=0; i<clientList.size(); i++)
+            {
+                if (strcasecmp(clientList[i].c_str(),client.c_str()) == 0)
+                {
+                    targetFound = true;
+                    break;
+                }
+            }
```

Comment:
Let's use `std::find(clientList.begin(), clientList.end(), client) != clientList.end()` instead.  Please ensure the indent spacing count matches the style of the method as well.